app/templates/fx_android/release/listing_apr_2016.php

```diff
@@ -102,8 +102,9 @@
 
 $whatsnew = function ($translations) use ($_) {
     return <<<OUT
-* {$_('A clean, modern visual refresh')}
-* {$_('An all-new home page with quick access to top sites and recent history')}
-* {$_('An experience designed in parallel with the new Firefox Quantum, our fast, modern desktop browser')}
+* {$_('Removed the Firefox Search widget from home screen')}
```

Comment:
Why did you change the order? This was the last element in Beta.

Reply:
For reference, this is the current order: https://l10n.mozilla-community.org/stores_l10n/locale/en-US/fx_android/beta/

Reply:
Yeah I realized why I got it wrong: I was both looking at that *and* the email thread we had with relman - which initally puts that string first. It should be good now just updated it

Reply:
Yep, order is good now.